### PR TITLE
MVC Querying updated for v8

### PR DESCRIPTION
**Pull request by poornimanayar**
Hi,

This is the PR for updated MVC Querying documentation for #1877 

Poornima

### CODE REVIEW

File: Reference/Templating/Mvc/querying.md [hunk 1]
```diff
@@ -59,92 +67,54 @@ Descendants(string nodeTypeAlias)
...
 ### Some examples
 
 #### Where children are visible
 
 ```csharp
-// dynamic access
-@CurrentPage.Children.Where("Visible")
-
-// strongly typed access
 @Model.Content.Children.Where(x => x.IsVisible())
```
Comment (sofietoft) [inline Reference/Templating/Mvc/querying.md]:
```suggestion
@Model.Children.Where(x => x.IsVisible())
```

File: Reference/Templating/Mvc/querying.md [hunk 2]
```diff
@@ -59,92 +67,54 @@ Descendants(string nodeTypeAlias)
...
 #### Traverse for sitemap
 
 ```csharp
-// dynamic access
-var values = new Dictionary<string,object>();
-values.Add("maxLevelForSitemap", 4);
-var items = @CurrentPage.Children.Where("Visible && Level <= maxLevelForSitemap", values);
-
-// strongly typed access
 var items = @Model.Content.Children.Where(x => x.IsVisible() && x.Level <= 4)
```
Comment (sofietoft) [inline Reference/Templating/Mvc/querying.md]:
```suggestion
var items = @Model.Children.Where(x => x.IsVisible() && x.Level <= 4)
```

File: Reference/Templating/Mvc/querying.md [hunk 3]
```diff
@@ -59,92 +67,54 @@ Descendants(string nodeTypeAlias)
...
 
 #### Content sub menu
 
 ```csharp
-// dynamic access
-// NOTE: you can also use NodeTypeAlias but is recommended to use DocumentTypeAlias
-@CurrentPage.AncestorOrSelf(1).Children.Where("DocumentTypeAlias == \"DatatypesFolder\"").First().Children
-
-// strongly typed
 @Model.Content.AncestorOrSelf(1).Children.Where(x => x.DocumentTypeAlias == "DatatypesFolder").First().Children
```
Comment (sofietoft) [inline Reference/Templating/Mvc/querying.md]:
```suggestion
@Model.AncestorOrSelf(1).Children.Where(x => x.DocumentTypeAlias == "DatatypesFolder").First().Children
```

File: Reference/Templating/Mvc/querying.md [hunk 4]
```diff
@@ -59,92 +67,54 @@ Descendants(string nodeTypeAlias)
...
-// dynamic access
-var paramVals = new Dictionary<string, object> {{"splitTerm", new char[] {','}}, {"searchId", "1173"}};
-var result = @CurrentPage.Ancestors().OrderBy("level")
-    .Single()
-    .Descendants()
-    .Where("selectedNodes != null && selectedNodes != String.Empty && selectedNodes.Split(splitTerm).Contains(searchId)", paramVals)
-    .FirstOrDefault();
 
-// strongly typed
 var result = @Model.Content.Ancestors().OrderBy(x => x.Level)
```
Comment (sofietoft) [inline Reference/Templating/Mvc/querying.md]:
```suggestion
var result = @Model.Ancestors().OrderBy(x => x.Level)
```

File: Reference/Templating/Mvc/querying-v7.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
+---
+versionFrom: 7.0.0
+needsV8Update: "true"
```
Comment (sofietoft) [inline Reference/Templating/Mvc/querying-v7.md]:
```suggestion

```

**Comment by poornimanayar** [comment]
Its just not my day isnt it @sofietoft LOL! Level 0 errors! Should be all fixed now. 

Poornima

**Comment by sofietoft** [comment]
😄 
No worries at all @poornimanayar !
Thanks for adding the suggestions 💪 

I'll get this one merged as well 🎉

**Pull request merged**